DEBUG - 2025-10-14 16:56:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 16:57:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 16:57:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 16:57:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 16:57:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 16:57:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 16:57:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 16:57:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 17:00:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:11:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:11:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:11:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:11:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:11:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:11:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:12:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:12:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:16:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:16:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:22 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '1')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:23 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '1')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:23 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'1\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'1\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '1')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:23 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'1\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'1\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'1\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '1')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:24 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '1')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:24 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '1')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:24 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'1\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'1\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '1')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:24 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'1\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'1\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'1\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '1')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:26 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '1')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:26 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '1')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:26 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'1\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'1\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '1')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:26 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'1\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'1\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'1\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '1')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:27 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '2')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:27 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '2')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:27 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '2')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:27 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:27 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '2')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:27 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '2')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:27 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '2')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:27 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:28 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '2')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:28 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '2')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:28 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'2\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '2')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:28 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'2\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'2\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'2\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '2')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:29 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '3')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '3')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:29 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'3\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '3')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:29 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'3\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'3\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '3')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:29 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '3')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '3')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:29 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'3\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '3')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:29 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'3\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'3\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'3\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '3')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:30 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '4')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:30 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '4')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:30 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'4\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'4\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '4')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:30 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'4\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'4\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'4\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '4')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:31 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:31 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:31 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:31 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:31 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:31 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:31 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:31 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:31 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:31 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:31 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:31 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:31 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:31 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:31 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:31 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:31 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:31 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:31 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:31 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:31 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:31 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:31 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:31 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:32 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:32 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:32 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:32 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'5\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '5')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:34 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '4')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '4')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:34 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'4\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'4\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '4')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:34 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'4\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'4\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'4\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '4')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:16:34 --> mysqli_sql_exception: Unknown column 'user_id' in 'where clause' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\ITE311-HERDA\app\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '4')
#7 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(933): App\Controllers\Course->enroll()
#8 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
#9 C:\xampp\htdocs\ITE311-HERDA\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\ITE311-HERDA\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-10-14 18:16:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
[Method: POST, Route: course/enroll]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 2 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 4 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '4')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:34 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'4\'')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'4\'')
 3 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 4 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 5 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 6 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '4')
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-10-14 18:16:34 --> [Caused by] mysqli_sql_exception: Unknown column 'user_id' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'4\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'4\'')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = \'3\'
AND `course_id` = \'4\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(1733): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `enrollments`
WHERE `user_id` = :user_id:
AND `course_id` = :course_id:', [...], false)
 5 SYSTEMPATH\Model.php(675): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 6 APPPATH\Models\EnrollmentModel.php(33): CodeIgniter\Model->countAllResults()
 7 APPPATH\Controllers\Course.php(33): App\Models\EnrollmentModel->isAlreadyEnrolled('3', '4')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Course->enroll()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Course))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:16:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-10-14 18:17:58 --> mysqli_sql_exception: Duplicate entry 'admin@lms.com' for key 'email' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Database\Seeds\UserSeeder.php(46): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-HERDA\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#7 C:\xampp\htdocs\ITE311-HERDA\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-HERDA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-HERDA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-HERDA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
ERROR - 2025-10-14 18:18:46 --> mysqli_sql_exception: Duplicate entry 'admin@lms.com' for key 'email' in C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\ITE311-HERDA\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(1841): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', NULL, false)
#4 C:\xampp\htdocs\ITE311-HERDA\system\Database\BaseBuilder.php(2230): CodeIgniter\Database\BaseBuilder->batchExecute('_insertBatch', 100)
#5 C:\xampp\htdocs\ITE311-HERDA\app\Database\Seeds\UserSeeder.php(46): CodeIgniter\Database\BaseBuilder->insertBatch(Array)
#6 C:\xampp\htdocs\ITE311-HERDA\system\Database\Seeder.php(149): App\Database\Seeds\UserSeeder->run()
#7 C:\xampp\htdocs\ITE311-HERDA\system\Commands\Database\Seed.php(79): CodeIgniter\Database\Seeder->call('App\\Database\\Se...')
#8 C:\xampp\htdocs\ITE311-HERDA\system\CLI\Commands.php(74): CodeIgniter\Commands\Database\Seed->run(Array)
#9 C:\xampp\htdocs\ITE311-HERDA\system\CLI\Console.php(47): CodeIgniter\CLI\Commands->run('db:seed', Array)
#10 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(388): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\ITE311-HERDA\system\Boot.php(133): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\ITE311-HERDA\spark(87): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}
DEBUG - 2025-10-14 18:21:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:21:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:22:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:22:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:22:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:22:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:22:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:22:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:22:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:22:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:22:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:22:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:22:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:22:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:22:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:22:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:22:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:23:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:23:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:23:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:23:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:23:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:37:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:37:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:37:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:41:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:45:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:45:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:45:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:45:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:45:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:48:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:48:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-10-14 18:48:27 --> Error: Class "App\Models\MaterialModel" not found
[Method: GET, Route: admin/course/1/upload]
in APPPATH\Controllers\Materials.php on line 35.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Materials->upload('1')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Materials))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:48:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:49:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:49:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:49:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:49:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:49:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:51:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 18:51:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-10-14 18:51:55 --> Error: Class "App\Models\MaterialModel" not found
[Method: GET, Route: admin/course/1/upload]
in APPPATH\Controllers\Materials.php on line 35.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Materials->upload('1')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Materials))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:53:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-10-14 18:53:50 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "materials/upload.php"
[Method: GET, Route: admin/course/1/upload]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('materials/upload.php')
 2 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('materials/upload', [], true)
 3 APPPATH\Controllers\Materials.php(39): view('materials/upload', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Materials->upload('1')
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Materials))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 18:53:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-10-14 18:53:51 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "materials/upload.php"
[Method: GET, Route: admin/course/1/upload]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('materials/upload.php')
 2 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('materials/upload', [], true)
 3 APPPATH\Controllers\Materials.php(39): view('materials/upload', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Materials->upload('1')
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Materials))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-10-14 19:02:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:02:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:02:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:03:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:03:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:03:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:03:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:03:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:03:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:05:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:05:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:05:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:05:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:06:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:06:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:06:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:06:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:06:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:06:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:06:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:06:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:06:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:06:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:06:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:08:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:08:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:08:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:08:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:09:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:09:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:09:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:10:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-10-14 19:10:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
